data/yassineatik.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Yassine Atik",
+  "bio": "Fullstack Web Developer : PHP | LARAVEL  ,  REACT | NEXT.JS",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SaraJaoude",
```

Comment:
Please provide your own twitter handle over here :)

Reply:
Thanks @RohanSasne , I have made the necessary changes .